### PR TITLE
chore: use probe-cli v3.19.0

### PR DESCRIPTION
**Pull request by aanorbel**
Part of https://github.com/ooni/probe/issues/2524

## Proposed Changes

  - Enable `echcheck` https://github.com/ooni/probe/issues/2567.
  - Add support for appending errors

### CODE REVIEW

**Comment by bassosimone** [comment]
@aanorbel it's 3.19.0-alpha

File: app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java [hunk 1]
```diff
@@ -43,11 +44,8 @@ public static CircumventionSuite initForAutoRun() {
                 list.add(new Psiphon());
             if (pm == null || pm.isTestTor())
                 list.add(new Tor());
-            /* TODO (aanorbel): Riseup VPN Disabled.
-                The riseupvpn experiment has been quite flaky for quite some time.
-                To be enabled only when test is fixed or removed if deemed necessary.
-                if (pm == null || pm.isTestRiseupVPN())
-                list.add(new RiseupVPN());*/
+            if (pm == null || pm.isTestRiseupVPN())
+                list.add(new RiseupVPN());
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java:48]: We should run RiseupVPN inside the experimental suite. If this change requires too much churn, I would recommend enabling it in the next release, but it must not run as part of circumvention.

**Review by bassosimone — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java (hunk 1 above)
Comment (aanorbel) [reply app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java]: Started a PR to effect this https://github.com/ooni/probe-android/pull/625 .

We would need to test properly and merge before the next release

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**